data/DALTON/basicDALTON-2015/Trp_polar_abalnr.dal

```diff
@@ -0,0 +1,63 @@
+BASIS
+STO-3G
+tryptophan static polarizability
```

Comment:
I was having a hard time getting DVB to converge during the polarizability step for some cases. I can try again and see which ones are troublesome.